README.md

```diff
@@ -182,11 +182,15 @@ If you find a security issue with our libraries or services please report it to
         </dict>
     </array>
 ```
-
 See more info about configuring redirect uri for MSAL in our [Wiki](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Redirect-uris-in-MSAL)
 
 Our library uses the ASWebAuthenticationSession for authentication on iOS 12 by default. See more information about default values, and support for other iOS versions [Wiki](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/MSAL-for-iOS-uses-web-browser)
 
+6. Add a new keychain group to your project Capabilities `com.microsoft.adalcache` . See more information about keychain groups for MSAL in our [Wiki](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS.)
```

Comment:
The "." in the end of the URL makes URL point to a non-existent page.
It should be (https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS) without dot.

Reply:
No problem ;)